### ISSUE TITLE
Objective-C BOSResource loadIfNeeded warning (warn_unused_result)

### ISSUE THREAD

**Issue by nderkach** [bustoutsolutions/siesta#230]
In Objective-C `loadIfNeeded` is expected to return a value and gives a warning:

`Ignoring return value of function declared with "warn_unused_result" attribute`

**Comment by pcantrell** [comment]
Yeah. I haven’t figured out how to suppress that; last I checked, `@discardableResult` just doesn’t translate to Obj-C. If you know how, I’d welcome a pull request!

In the meantime, you can suppress the warning like this:

```Objective-C
_ = [resource loadIfNeeded];
```
